src/Cache/Engine/ApcEngine.php

```diff
@@ -181,9 +180,8 @@ public function add($key, $value)
         if ($duration) {
             $expires = time() + $duration;
         }
-        apcu_add($key . '_expires', $expires, $duration);
 
-        return apcu_add($key, $value, $duration);
+        return apcu_add($key, $value, $duration) === true && apcu_add($key . '_expires', $expires, $duration);
```

Comment:
I think the return checking is a good idea. Not a fan of the really long line.

Reply:
Well the long line was just cause I did it quickly through github.